Resources/Private/Partials/Search/AddSearchFieldDialog.html

```diff
@@ -0,0 +1,115 @@
+<f:comment>
+    <!--
+    This file is part of the TYPO3 CMS project.
+
+    It is free software; you can redistribute it and/or modify it under
+    the terms of the GNU General Public License, either version 2
+    of the License, or any later version.
+
+    For the full copyright and license information, please read the
+    LICENSE.txt file that was distributed with this source code.
+
+    The TYPO3 project - inspiring people to share!
+    -->
+</f:comment>
+
+<div class="modal" id="add-searchfield-dialog" tabindex="-1" role="dialog">
```

Comment:
Are the values in this HTML template not translated?

Reply:
The missing translations have been added.